doc/API.md

```diff
@@ -431,6 +432,30 @@ None; the `result` field will be set to the empty object `{}`. Any value should
 disregarded for forward compatibility.
 
 
+### `getserverstatus`
+
+Retrieve the status of the servers, such as the coordinator, the cosigners, the watchtowers
+
+#### Request
+
+| Field          | Type   | Description                                    |
+| -------------- | ------ | ---------------------------------------------- |
+
+#### Response
+
+| Field          | Type   | Description                                         |
+| -------------- | ------ | --------------------------------------------------- |
+| `coordinator`  | object | [Server status](#server-status) for the coordinator |
+| `cosigners`    | array  | Array of [Server status](#server-status)            |
+| `watchtowers`  | array  | Array of [Server status](#server-status)            |
+
+##### Server status
+
+| Field    | Type   | Description                                                 |
+| -------- | ------ | ----------------------------------------------------------- |
+| `status` | string | Status of the server, can be `reachable`, `not_reachable`   |
```

Comment:
If this is a string for forward compatibility, we can just add new entries to the object without breaking parsing for previous versions. I think it should only be a `reachable` bool.

Reply:
I was thinking that in the future we could go with more detailed status, such as "it's dead", "it's alive but I can't complete the handshake", "it's alive and we can chit-chat without troubles"

Reply:
Then it would be an API break. Adding fields to an object isn't

Reply:
Ah, good point! I'll update the code